### ISSUE TITLE
[Request] Desktop zoom improvement

### ISSUE THREAD

**Issue by shmink** [linuxmint/cinnamon#10084]
Apologies if this is in the wrong repo.

Me and a couple of my friends, all on mint 20.1 and on cinnamon 4.8.6 use desktop zoom a lot coming from macbooks.

The built in desktop zoom in the accessibility menu does the job but it's not really useable. If your mouse is hovering over something that can scroll then it will do as you do your zoom. 

For example, you're in your browser reading an article and you want to use desktop zoom to get closer. Hold down <kbd>Alt</kbd> + scroll but now you've scrolled away from the area you wanted to look at.

Lens mode helps a bit as at least the areas of the screen not zooming in are in your peripheral and you can see the page moving around with the scroll.

**Comment by mtwebster** [comment]
I can only reproduce this with chromium (and other chromium-based things, like slack, which is based on it).  Firefox is ok, as are any normal programs I tried (file manager, terminal,etc..).

What other programs does it do this on? It seems like maybe a chromium issue.

**Comment by shmink** [comment]
You're right. There's just so many chromium based apps that I didn't notice the connection. Slack, WhatsApp desktop and chrome are all moving when I try and zoom. Terminal and Firefox are fine.

Is this a problem that can only be addressed in chromium then?

**Comment by BillyRayPreachersSon** [comment]
As mentioned in #10148, it also scrolls built-in apps, such as the System Settings app.

Apologies, if I'd found this issue, I wouldn't have opened #10148.
